biothings/hub/dataplugin/assistant.py

```diff
@@ -302,24 +302,35 @@ def get_uploader_dynamic_class(self, uploader_section, metadata, sub_source_name
                         import orjson
 
                         # Setup parser to parser factory
-                        from {mod} import {func} as parser_factory
+                        from {mod} import {func} as parser_func
 
                         parser_kwargs_serialized = r\'\'\'
                             {parser_kwargs_serialized}
                         \'\'\'  # I am not 100 percent certain this works
 
                         parser_kwargs = orjson.loads(parser_kwargs_serialized)
-                        parser_func = parser_factory(**parser_kwargs)
                     '''
                     )
                 else:
+                    # create empty parser_kwargs to pass to parser_func
+                    parser_kwargs_serialized = orjson.dumps({}).decode('utf-8')
+
                     confdict["PARSER_FACTORY_CODE"] = textwrap.dedent(
                         f'''
+                    # get json
+                    import orjson
+
                     # when code is exported, import becomes relative
                     try:
                         from {self.plugin_name}.{mod} import {func} as parser_func
                     except ImportError:
                         from .{mod} import {func} as parser_func
+
+                    parser_kwargs_serialized = r\'\'\'
+                        {parser_kwargs_serialized}
+                    \'\'\'  # I am not 100 percent certain this works
+
+                    parser_kwargs = orjson.loads(parser_kwargs_serialized)
```

Comment:
@sengineer0 is `repr` a better way to get the string representation of an object?

Reply:
not sure, let's me check it first

Reply:
@newgene  it's good idea. I tested and it worked well. I also updated and pushed the new change.
Please take a look, thanks :)